src/bootstrap/bin/rustdoc.rs

```diff
@@ -24,14 +24,10 @@ fn main() {
     let mut dylib_path = bootstrap::util::dylib_path();
     dylib_path.insert(0, PathBuf::from(libdir.clone()));
 
-    //FIXME(misdreavus): once stdsimd uses cfg(doc) instead of cfg(dox), remove the `--cfg dox`
-    //arguments here
     let mut cmd = Command::new(rustdoc);
     cmd.args(&args)
         .arg("--cfg")
         .arg(format!("stage{}", stage))
```

Comment:
Hm, this also looks stale, we should really be passing --cfg bootstrap in stage 0 these days, cfg(stage0) shouldn't be used by anything.

Want to update that in this PR as well? (We should match rustc.rs).

Reply:
It looks like `rustc.rs` doesn't set it either, I did find it set in `compile.rs` though:

```
compile.rs
422-                        .arg("--cfg")
423:                        .arg("bootstrap")
```
Maybe nothing actually needs `--cfg bootstrap` for rustdoc? I'll try removing `--cfg stage0` and see.